src/streaming/net/XHRLoader.js

```diff
@@ -65,7 +65,9 @@ function XHRLoader(cfg) {
         }
 
         if (requestModifier) {
-            xhr = requestModifier.modifyRequestHeader(xhr);
+            xhr = requestModifier.modifyRequestHeader(xhr, {
+                url: httpRequest.url
+            });
         }
```

Comment:
Please also add this parameter to the sample page in `samples/advanced/extend.html`